packages/api/es/search.js

```diff
@@ -67,19 +67,47 @@ const esProdConfig = async (host) => {
   };
 };
 
-const esConfig = (host) => (inTestMode() ? esTestConfig() : esProdConfig(host));
+const esMetricsConfig = () => {
```

Comment:
This should probably have a unit test on the new logic....admittedly it looks like the unit testing on this file isn't present, which is it's own seperate problem.   🤔

Reply:
Added a test file, but only added tests for testing the config for now. This class (and it's lack of tests), is a candidate for some IP work, IMO

Reply:
That sounds good.    We should add a ticket if there isn't one already.      I can take a stab at it, will post here if/when done.

Reply:
Created [CUMULUS-1747](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1747)